check_db_genesis_balances.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+from dataclasses import dataclass
+from typing import Optional, TypedDict
+import psycopg2
+import json
+import os
+import argparse
+
+name = os.environ.get("DB_NAME", "archive")
+user = os.environ.get("DB_USER", "mina")
+password = os.environ.get("DB_PASSWORD")
+host = os.environ.get("DB_HOST", "localhost")
+port = os.environ.get("DB_PORT", "5432")
+config_file = os.environ.get("CONFIG_FILE")
+
+
+parser = argparse.ArgumentParser(
+    description="Check that the genesis balances in the JSON file match the database."
+)
+parser.add_argument(
+    "--config-file",
```

Comment:
NIT: config file is to open name. Do you mean ledger json or replayer config file?